setup.cfg

```diff
@@ -62,8 +62,8 @@ visual_test =
     jupyterlab~=3.0
     bqplot
     scipy
-    ipympl==0.8.7
-    ipyvolume
+    ipympl==0.9.2
+    ipyvolume @ git+https://github.com/jtpio/ipyvolume@ipywidgets-8
```

Comment:
cc @maartenbreddels it would be great to have a release of `ipyvolume` with support for `ipywidgets` 8: https://github.com/widgetti/ipyvolume/pull/411, thanks!